nuget/targets/netstandard/Microsoft.ML.OnnxRuntime.Extensions.props

```diff
@@ -1,17 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
-  <ItemDefinitionGroup Condition="'$(PlatformTarget)' == 'x64' OR ('$(PlatformTarget)' == 'AnyCPU' AND '$(Prefer32Bit)' != 'true')">
-	<Link>
-	  <AdditionalDependencies>$(MSBuildThisFileDirectory)../../runtimes/win-x64/native/onnxruntime.lib;%(AdditionalDependencies)</AdditionalDependencies>
-	</Link>
-  </ItemDefinitionGroup>
-
-  <ItemDefinitionGroup Condition="'$(PlatformTarget)' == 'x86' OR ('$(PlatformTarget)' == 'AnyCPU' AND '$(Prefer32Bit)' == 'true')">
-	<Link>
-	  <AdditionalDependencies>$(MSBuildThisFileDirectory)../../runtimes/win-x86/native/onnxruntime.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
should it be ortextensions.lib?

Reply:
I believe RelWithDebInfo does not have a ortextensions.lib, that was in Release. Regardless, this was the old reference to the ort lib which we removed from the package and so we are removing it from the props file too.